packages/api/lib/FileUtils.js

```diff
@@ -85,21 +89,26 @@ const buildDatabaseFile = (providerURL, file) =>
     setChecksum,
     setFileName,
     partial(setSource, providerURL),
-    filterDatabaseProperties,
-    removeNilProperties,
     setS3FileSize // This one is last because it returns a Promise
   ])(file);
 
+const cleanDatabaseFile = (file) =>
```

Comment:
Would be nice to have a unit test for this function and even `buildDatabaseFiles`

Reply:
Both are tested in `api/tests/models/test-granules-model.js`. See the various `new schema` tests, which will have these functions in the call stack.